### PR TITLE
question 5

### PR DESCRIPTION
**Pull request by Isaac-Madden**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -101,3 +101,51 @@ describe("/api/articles/:article_id", () => {
...
+        expect(data.body.articles).toBeSortedBy("created_at", { descending: true });
+    })
+  })
+
+  test("each article object needs specified properties. There should not be a body property", () => {
+      return request(app)
+      .get("/api/articles")
+      .then((data) => {
+          data.body.articles.forEach( (article) => {
+              expect(article.hasOwnProperty('author')).toBe(true);
```
Comment (two-triangles) [inline __tests__/app.test.js]: This test could be made more thorough by checking the type of the property value.

File: Server/models/articles.model.js [hunk 2]
```diff
@@ -12,4 +12,20 @@ exports.fetchArticleByID = (article_id) => {
         else return data.rows[0];
         
     });
+};
+
+exports.fetchAllArticles = () => {
+
+    const queryString = `SELECT articles.author, title, articles.article_id, topic, articles.created_at, articles.votes, article_img_url, 
+    COUNT(comments.article_id) AS comment_count
```
Comment (two-triangles) [inline Server/models/articles.model.js]: Just be aware, that your comment_count will come back as a string and not an integer.

File: __tests__/app.test.js (hunk 1 above)
Comment (Isaac-Madden) [reply __tests__/app.test.js]: updated!

File: Server/models/articles.model.js (hunk 2 above)
Comment (Isaac-Madden) [reply Server/models/articles.model.js]: thanks for the heads up :)

**Review by Beth-McCormack — APPROVED** [review]

**Pull request merged**